client/signup/style.scss

```diff
@@ -61,3 +61,22 @@
 .is-section-signup .layout__content {
 	overflow: visible;
 }
+
+/* New Signup Styles */
+
+.formatted-header__title {
+	font-family: $signup-sans;
+	font-size: 48px;
```

Comment:
General question, that shouldn't get resolved in this PR, have we ever revisited our measuring unit (pixels) for type faces? It seems a little antiquated to be using pixels for text (or anything in general)

Reply:
Oh, but that's a good point. Looking at /devdocs we should use the font sizes corresponding to the Perfect Fifth scale, which has corresponding `em` values. I'll swap those out.